prereise/cli/data_sources/demand_data.py

```diff
@@ -0,0 +1,32 @@
+from prereise.cli.data_sources.data_source import DataSource
+from prereise.cli.data_sources.exceptions import CommandNotSupportedError
+
+
+class DemandData(DataSource):
+    @property
+    def command_name(self):
```

Comment:
This could optionally be simplified even more by defining `class NotSupportedDataSource(DataSource)`, then having demand and hydro inherit from that and override `command_name`.

Reply:
good point. updated, PTAL when you get the chance